camayoc/tests/qcs/api/v1/credentials/test_network_creds.py

```diff
@@ -180,15 +183,50 @@ def test_negative_create_key_and_pass(cleanup, isolated_filesystem):
     assert cred._id is None
 
 
-@pytest.mark.skip
-def test_create_sudo_password(cleanup, isolated_filesystem):
-    """Create a network credential that has a sudo password.
+@pytest.mark.parametrize('method', QCS_BECOME_METHODS)
+def test_create_become_method(cleanup, shared_client, method):
+    """Create a network credential that uses become options.
 
     :id: e49e497d-abb7-4d6a-8366-3409e297062a
-    :description: Create a network credential with username, password XOR
-        sshkey, and a sudo password.
+    :description: Create a network credential with username, password
+        and uses a become method.
+    :steps: Send a POST to the credential endpoint with data.
+    :expectedresults: A new network credential is created.
+    :caseautomation: notautomated
+    """
+    cred = Credential(
+        cred_type='network',
+        client=shared_client,
+        password=uuid4(),
+        become_method=method,
+        become_password=uuid4(),
+        become_user=uuid4(),
+    )
+    cred.create()
+    # add the id to the list to destroy after the test is done
+    cleanup.append(cred)
+    assert_matches_server(cred)
+
+
+@pytest.mark.parametrize('fake_method', ['not-a-method', 86])
+def test_create_negative_method(cleanup, shared_client, fake_method):
+    """Create a network credential that uses become options.
+
+    :id: f05f2ea8-ae9f-4bad-a76f-5246128400d9
+    :description: Create a network credential with username, password
+        and uses a become method.
```

Comment:
I think here you are not trying a become method, but an invalid one, right? Can you please update the description?